src/routes/Player/ControlBar/SeekBar/SeekBar.js

```diff
@@ -56,7 +63,19 @@ const SeekBar = ({ className, time, duration, buffered, onSeekRequested }) => {
                 onSlide={onSlide}
                 onComplete={onComplete}
             />
-            <div className={styles['label']}>{formatTime(duration)}</div>
+            <Button
+                onClick={() => {
+                    durationMode === DURATION_MODE_OPTIONS.totalTime
+                        ? setDurationMode(DURATION_MODE_OPTIONS.remainingTime)
+                        : setDurationMode(DURATION_MODE_OPTIONS.totalTime);
+                }}
+            >
+                <div className={styles['label']}>
+                    {durationMode === DURATION_MODE_OPTIONS.totalTime
```

Comment:
Also this `remainingTimeMode` instead of `durationMode === DURATION_MODE_OPTIONS.totalTime`